likecoin/js/admin-settings/src/components/Publish/InternetArchive/InternetArchiveDescription.js

```diff
@@ -29,9 +29,9 @@ function InternetArchiveDescription() {
   );
   return (
     <div>
-      <h2>
+      <h3>
         {__('What is Internet Archive (archive.org)?', 'likecoin')}
-      </h2>
+      </h3>
```

Comment:
I think can remove this header
![image](https://user-images.githubusercontent.com/17264716/207302784-7f5bafd8-d7c9-430e-8f33-704fa075ed6a.png)

Reply:
updated